.changeset/four-panthers-flash.md

```diff
@@ -0,0 +1,10 @@
+---
+"@cube-creator/core-api": patch
+---
+
+Added new fields to cube metadata:
+ - `Planned update https://schema.ld.admin.ch/datasetNextDateModified>`
```

Comment:
I'm not particularly happy with putting the `https://schema.ld.admin.ch/` namespace directly into code but I don't have a good idea for a better option